src/libs/NetworkConnection.js

```diff
@@ -1,10 +1,19 @@
 import _ from 'underscore';
+import Onyx from 'react-native-onyx';
 import NetInfo from '@react-native-community/netinfo';
 import AppStateMonitor from './AppStateMonitor';
 import Log from './Log';
 import * as NetworkActions from './actions/Network';
 import CONFIG from '../CONFIG';
 import CONST from '../CONST';
+import ONYXKEYS from '../ONYXKEYS';
+
+let shouldForceOffline;
+Onyx.connect({
+    key: ONYXKEYS.NETWORK,
+    waitForCollectionCallback: true,
+    callback: val => shouldForceOffline = val.shouldForceOffline || false,
```

Comment:
NAB, could be consistent with our use of `Boolean()` here as well

```suggestion
    callback: val => shouldForceOffline = Boolean(val.shouldForceOffline),
```